go.mod

```diff
@@ -10,36 +10,71 @@ require (
 	github.com/charmbracelet/lipgloss v0.5.0
 	github.com/hako/durafmt v0.0.0-20210608085754-5c1018a4e16b
 	github.com/joho/godotenv v1.4.0
+	github.com/lucasepe/codename v0.2.0
 	github.com/matryer/moq v0.2.7
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.4.0
 	github.com/zalando/go-keyring v0.2.1
 	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
 	golang.org/x/term v0.0.0-20220526004731-065cf7ba2467
 	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.24.1
+	k8s.io/apimachinery v0.24.1
+	k8s.io/client-go v0.24.1
```

Comment:
There is something important about compatibility.
https://github.com/kubernetes/client-go#compatibility-matrix